keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,44 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes):
+    # padded ragged bounding box tensor with -1s to create a tensor
+    # This enables vectorized map execution
+    return bounding_boxes.to_tensor(-1)
+
+
+def filter_sentinels(bounding_boxes):
+    # filter the padded bounding boxes
+    def is_sentinel(box):
+        return tf.cond(box[4] == -1, lambda: False, lambda: True)
```

Comment:
I think you can remove the tf.cond and just write box[4] == -1

Reply:
Done!